examples/trials/mnist-hyperband/config.yml

```diff
@@ -0,0 +1,24 @@
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 2
+maxExecDuration: 100h
+maxTrialNum: 10000
+#choice: local, remote, pai
+trainingServicePlatform: local
+searchSpacePath: search_space.json
+#choice: true, false
+useAnnotation: false
+advisor:
+  #choice: Hyperband
+  builtinAdvisorName: Hyperband
+  classArgs:
+    #R: the maximum STEPS
```

Comment:
maximum step for what?

Reply:
this is not straightforward, please refer to our doc of hyperband

Reply:
fixed, thx